Gemfile

```diff
@@ -39,7 +39,7 @@ gem "gettext_i18n_rails_js",          "~>1.1.0"
 gem "hamlit",                         "~>2.7.0"
 gem "highline",                       "~>1.6.21",      :require => false
 gem "inifile",                        "~>3.0",         :require => false
-gem "kubeclient",                     "~>2.4.0",       :require => false # For scaling pods at runtime
+gem "kubeclient",                     "~>2.5.2",       :require => false # For scaling pods at runtime
```

Comment:
@carbonin can we loosen this dependency allowing the other repos to be upgraded?

Reply:
How would you want to declare it? `~>2.5`?

Reply:
Or rather maybe `~>2.4` then we shouldn't have to merge anything when red.